en/homework/README.md

```diff
@@ -134,7 +134,7 @@ just under a line with the edit button.
 Now we need a URL (`blog/urls.py`):
 
 ```python
-path('post/<pk>/remove/', views.post_remove, name='post_remove'),
+path(post/<pk>)/remove/', views.post_remove, name='post_remove'),
```

Comment:
Here! Found it. You fixed the missing quotation mark at the beginning of the link, but also added a close parentheses after <pk> that shouldn't be there. It's already at the end of the line!
So it should be:
`path(post/<pk>/remove/', views.post_remove, name='post_remove'),`

Reply:
Changing your base branch to master (it was somehow changed to revert-1-master) will fix this.